cmd/deploy.go

```diff
@@ -12,13 +12,27 @@ import (
 )
 
 var (
+	deployExample = `
+# Deployment you would like to deploy to Airflow cluster:
+astro deploy physical-diameter-1566
```

Comment:
@andriisoldatenko what do you think about using some kind of placeholder, instead of a fake deployment name here?

Reply:
@schnie do you mean like this?
`$ astro deploy <deployment name>`

Reply:
yep

Reply:
done.